pkg/yurthub/cachemanager/cache_restmapper.go

```diff
@@ -0,0 +1,110 @@
+/*
+Copyright 2020 The OpenYurt Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cachemanager
+
+import (
+	normaljson "encoding/json"
+	"strings"
+
+	"github.com/openyurtio/openyurt/pkg/yurthub/kubernetes/serializer"
+
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/klog"
+)
+
+var (
+	cacheDynamicRESTMapperKey = "_internal/cache-manager/cache-crd-restmapper.conf"
+	sepForGVR                 = "/"
+)
+
+func (cm *cacheManager) initRESTMapperManager() {
```

Comment:
In my opinion, `RESTMapper` info is not part of cacheManager, so it's not friendly to add func of `RESTMapper` in cacheManager. how about remove cache of restMapper into `serializer.go`?

Reply:
Thanks for the suggestion, the code has been updated.